### DISCUSSION TITLE
Is it possible to run lighthouse as kubernetes container?

### DISCUSSION THREAD

**Discussion by SenthilkumarRasan** [GoogleChrome/lighthouse#15404] [General]
**Summary**
Can we run lighthouse as kubernetes container for better scalability?
If so is there sample config for deployment?

**Comment by adamraine** [comment]
Closest thing we have is an example docker configuration for Lighthouse CI: https://github.com/GoogleChrome/lighthouse-ci/tree/main/docs/recipes/docker-client

There's probably a way to use kubernetes but we don't have a sample config unfortunately. Perhaps someone in the community who has tried this can chime in.
